src/components/DisplayPages/DisplayPageMain.js

```diff
@@ -16,7 +20,46 @@ export default function DisplayPageMain({ masterVolume, onChangeInputVolume }) {
           defaultValue={masterVolume}
           onChange={onChangeInputVolume}
         />
-      </label>
+      </label> */}
```

Comment:
Whats happening btw. line 13 and 23 ? Kind of commented out ... Maybe something to delete?